### PR TITLE
Change tracker

### PR DESCRIPTION
**Pull request by tuube**
Created a plugin for saving content updates. Useful for content creators

### CODE REVIEW

File: config/server.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 module.exports = ({ env }) => ({
-  host: env('HOST', '0.0.0.0'),
-  port: env.int('PORT', 1337),
+  host: env("HOST", "0.0.0.0"),
+  port: env.int("PORT", 1337),
   admin: {
     auth: {
-      secret: env('ADMIN_JWT_SECRET', '255a8bb6c62d421ce19160f15bea8aa9'),
+      secret: env("ADMIN_JWT_SECRET", "255a8bb6c62d421ce19160f15bea8aa9"),
```
Comment (RikuRuokonen) [inline config/server.js:6]: Should these be configured with Azure Secrets(or similar, called secrets in aws)

File: plugins/change-tracker/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Strapi plugin change-tracker
+
+A custom plugin for tracking which content changes are published and which aren't.
+This plugin works by writing all content changes to db, and then it compares their creation times to the last deployment's time.
+
+Changes ovr a month old are deleted automatically.
```
Comment (RikuRuokonen) [inline plugins/change-tracker/README.md:6]: Does this mean that plugin deletes unpublished changes after month? If so, I would leave the deleting of content solely to authors instead of forcing it via code.

File: config/server.js (hunk 1 above)
Comment (tuube) [reply config/server.js:6]: This takes the value by default from the env variable ADMIN_JWT_SECRET, which is defined as an Application setting in Azure Web App, and I think those are fine for this use case:

> Application settings are encrypted at rest and transmitted over an encrypted channel. You can choose to display them in plain text in your browser by using the controls below. Application Settings are exposed as environment variables for access by your application at runtime.

File: plugins/change-tracker/README.md (hunk 2 above)
Comment (tuube) [reply plugins/change-tracker/README.md:6]: Oh bad wording by on my part. Meant to say that the change-data, not the content itself, is deleted after one month. But I could add logic that only deployed changes older than one month are deleted. 

TL;DR; This doesn't affect changed content in anyway, only the data about the change (that we ourself gather).

**Review by RikuRuokonen — APPROVED** [review]
